examples/qip-scheduler.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>Although parallelly is a word, its usage is not advised. "in parallel" is better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>